dao/src/main/java/com/fit_goal/util/EventMongoConverter.java

```diff
@@ -0,0 +1,29 @@
+package com.fit_goal.util;
+
+import com.fit_goal.domain.EventDto;
+import lombok.experimental.UtilityClass;
+import org.bson.Document;
+
+import java.time.ZoneId;
+
+import static com.fit_goal.EventDtoFields.*;
+
+
+@UtilityClass
+public class EventMongoConverter {
+
+    public EventDto documentToEventDto(Document document) {
+        EventDto eventDto = new EventDto();
+        eventDto.setId(document.getObjectId(ID));
+        eventDto.setEvent(document.getString(EVENT));
+        eventDto.setServiceName(document.getString(SERVICE_NAME));
+        eventDto.setDate(document.getDate(DATE).toInstant().atZone(ZoneId.systemDefault()).toLocalDateTime());
```

Comment:
it's hard to read this chain of calls, extract into small private method and format the code

Reply:
refactored